frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -903,6 +922,83 @@ function ReaderPaging:onGotoPageRel(diff)
             return true
         end
 
+    elseif self.zoom_mode:find("pan") then
+        -- pan zoom mode
+        logger.dbg("ZOOM: Pan mode")
+
+        local panning_update = {x = nil, y = nil}
+        local page_area, old_va = self.page_area, self.visible_area
+        local x, y, w, h, left = "x", "y", "w", "h", "left"
```

Comment:
Seems unexpected to me, bad for future skimming.
```suggestion
        local x, y, w, h
        local left = "left"
```

Reply:
Do you mean:
```lua local x, y, w, h = "x", "y", "w", "h"``` ?

This is necessary to handle the "column first" case without doubling the number of lines.

Reply:
Yes.